### ISSUE TITLE
Support for Angular 17

### ISSUE THREAD

**Issue by jandeu** [sweetalert2/ngx-sweetalert2#255]
Please update dependencies to support Angular 17 

```
npm ERR! Could not resolve dependency:
npm ERR! peer @angular/common@"^14.0.0 || ^15.0.0 || ^16.0.0" from @sweetalert2/ngx-sweetalert2@12.2.0
npm ERR! node_modules/@sweetalert2/ngx-sweetalert2
npm ERR!   @sweetalert2/ngx-sweetalert2@"^12.2.0" from the root project
```

**Comment by jandeu** [comment]
There is already opened PR https://github.com/sweetalert2/ngx-sweetalert2/pull/252

**Comment by ivanvastag** [comment]
Will this get merged any time soon?

**Comment by FGO-LPCR** [comment]
Any news on this issue? @limonte @toverux @zenflow

**Comment by D-Biela** [comment]
Any news?

**Comment by Lucasrsv1** [comment]
Until the PR is merged, we can workaround this error by adding the following override to `package.json`:

```json
"overrides": {
	"@sweetalert2/ngx-sweetalert2": {
		"@angular/common": "^14.0.0 || ^15.0.0 || ^16.0.0 || ^17.0.0",
		"@angular/core": "^14.0.0 || ^15.0.0 || ^16.0.0 || ^17.0.0"
	}
}
```

I hope this helps.

**Comment by limonte** [comment]
:tada: This issue is solved in version 12.3.0 :tada:

The release is available on:
- [npm package (@latest dist-tag)](https://www.npmjs.com/package/@sweetalert2/ngx-sweetalert2/v/12.3.0)
- [GitHub release](https://github.com/sweetalert2/ngx-sweetalert2/releases/tag/v12.3.0)

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:
